packages/react/src/components/NoticeChoice/__stories__/NoticeChoice.stories.js

```diff
@@ -0,0 +1,113 @@
+/**
```

Comment:
Lots of comments in this file that can be removed.

Reply:
Already mentioned up top by @annawen1. Lots of comments in this file that can be removed.